### PR TITLE
[WEBSITE-1542][WEBSITE-1494] Show page redesign, postcode new design, structural changes

### PR DESCRIPTION
**Pull request by cwaszczuk**

### CODE REVIEW

File: src/javascripts/dropdown.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	}
+
+	// Add / Remove hidden class
+	function toggleDropdown(){
+		// Get parent of dropdown
+		dropdown_parent = this.parentElement;
+		dropdown_content = dropdown_parent.querySelector('.dropdown__content');
+
+		//Check dropdown content
+		if (dropdown_content.className.includes('hidden')) {
```
Comment (UsmanAfzal) [inline src/javascripts/dropdown.js]: The `includes` method is not supported below IE12. Alternative would be to use `classList.contains('hidden')`

File: src/templates/prototypes/people-id.pug [hunk 2]
```diff
@@ -3,11 +3,41 @@ block content
...
+					<dd>
+						<a href="mailto:nigel.adams.mp@parliament.uk">nigel.adams.mp@parliament.uk</a>
+					</dd>
+					<dt>Phone: </dt>
+					<dd>
+						<a href="tel:02072197141">020 7219 7141</a>
+					</dd>
+					<dt> Address: </dt>
+					<dd>House of Commons, London, SW1A 0AA</dd>
+				</dl>
```
Comment (UsmanAfzal) [inline src/templates/prototypes/people-id.pug]: Need to update from HTML to pug

File: src/templates/prototypes/people-id.pug [hunk 3]
```diff
@@ -3,11 +3,41 @@ block content
 	.section--primary(id="content")
 		.container
 			.block--border__bottom
-				.section--primary__container
-					+image('../../images/diane-abbott.jpg', 'Dianne Abbott')
-					.section--primary__block
-						h1 Diane Abbott
-						p #[+link('#', 'Labour')] MP for #[+link('#', 'Hackney North and Stoke Newington')]
+					+image('../../images/kenneth_clark_1464x584.jpg', 'Dianne Abbott')
```
Comment (UsmanAfzal) [inline src/templates/prototypes/people-id.pug]: image path would have changed now to /images

File: src/stylesheets/objects/_content-layouts.scss [hunk 4]
```diff
@@ -4,9 +4,18 @@
 
 .section--primary {
 	padding-top: $base-spacing-unit-extra-large;
+
+	+ section > .container > .inner-container--focus {
```
Comment (UsmanAfzal) [inline src/stylesheets/objects/_content-layouts.scss]: Don't think we are using this any longer? Replaced with blocks?

File: src/javascripts/dropdown.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+// Opens respective dropdown when clicked
+(function () {
+
+	// Gather elements
+	dropdown_toggle = document.getElementsByClassName('dropdown__toggle');
```
Comment (UsmanAfzal) [inline src/javascripts/dropdown.js]: Need to define `dropdown_toggle` by including `var`

File: src/javascripts/dropdown.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+	// Add listener to dropdowns
+	for (var i = 0; i < dropdown_toggle.length; i++) {
+		dropdown_toggle[i].addEventListener('click', toggleDropdown, false);
+	}
+
+	// Add / Remove hidden class
+	function toggleDropdown(){
+		// Get parent of dropdown
+		dropdown_parent = this.parentElement;
+		dropdown_content = dropdown_parent.querySelector('.dropdown__content');
```
Comment (UsmanAfzal) [inline src/javascripts/dropdown.js]: Define `dropdown_parent` and `dropdown_content` by adding `var`

File: src/javascripts/hasJS.js [hunk 7]
```diff
@@ -5,4 +5,10 @@
 	// switches 'no-js' to 'has-js' class on body
 	document.body.className = 'has-js';
 
+	// Hide dropdowns if JS is on
+	dropdown_content = document.getElementsByClassName('dropdown__content');
```
Comment (UsmanAfzal) [inline src/javascripts/hasJS.js]: Define `dropdown_content` by adding `var`

File: src/templates/prototypes/people-id.pug (hunk 3 above)
Comment (cwaszczuk) [inline src/templates/prototypes/people-id.pug]: Not sure why this would be the case? Still sits 2 directories in while in the public folder.

**Pull request merged**